viroconcom/plot.py

```diff
@@ -0,0 +1,520 @@
+import numpy as np

```

Comment:
It seems to me, that this module does not belong in viroconcom as it does something very different. While viroconcom computes the contours this module rather uses viroconcom to work with the already created contours.

How about creating a new package e.g. viroconplot in the top level of this repo? This way the tests for viroconplot can always use the up-to-date viroconcom version, but we reduce the coupling of the two and split them semantically.

Reply:
Let's discuss this during a video call. 

At the moment I feel that having it as part of viroconcom is better as having another package with another version number adds complexity. Should users of viroconcom need to install two packages using pip then?